src/fastcs/attributes.py

```diff
@@ -28,18 +34,29 @@ async def update(self, controller: Any, attr: AttrR) -> None:
 
 @runtime_checkable
 class Handler(Sender, Updater, Protocol):
+    """Protocol encapsulating both `Sender` and `Updater`."""
+
     pass
 
 
 class Attribute(Generic[T]):
+    """Base FastCS attribute.
+
+    Instances of this class added to a `Controller` will be exposed to the backend.
+    """
+
     def __init__(
-        self, datatype: DataType[T], access_mode: AttrMode, handler: Any = None
+        self,
+        datatype: DataType[T],
+        access_mode: AttrMode,
+        group: Optional[str] = None,
```

Comment:
Use `<type> | None` for Optional, as done elsewhere in this file.

There is no need for `Optional` in this PR.